src/main/java/com/dabsquared/gitlabjenkins/trigger/handler/push/OpenMergeRequestPushHookTriggerHandler.java

```diff
@@ -146,7 +146,7 @@ private void handleMergeRequest(
             List<Action> actions = Arrays.<Action>asList(
                     new CauseAction(new GitLabWebHookCause(retrieveCauseData(hook, project, mergeRequest, branch))),
                     new RevisionParameterAction(commit, retrieveUrIish(hook)));
-            scheduleBuild(job, actions.toArray(new Action[actions.size()]));
+            scheduleBuild(job, actions.toArray(new Action[0]));
```

Comment:
I think we will have to undo this change as an Array is of fixed size unlike an ArrayList in Java.